app/views/content/landing/how-much-do-teachers-get-paid/_collage.html.erb

```diff
@@ -13,7 +13,7 @@
             "The average salary for a classroom teacher in England is around £39k a year."
           ],
           image_paths: [
-            "media/images/content/how-much-do-teachers-get-paid/collage3.jpg",
+            "media/images/content/how-much-do-teachers-get-paid/collage4.jpg",
```

Comment:
this image looks a bit big @philspray - does it need resizing (not sure how that works!)?